Sources/Optimizely/OptimizelyJSON.swift

```diff
@@ -0,0 +1,143 @@
+/****************************************************************************
+* Copyright 2020, Optimizely, Inc. and contributors                        *
+*                                                                          *
+* Licensed under the Apache License, Version 2.0 (the "License");          *
+* you may not use this file except in compliance with the License.         *
+* You may obtain a copy of the License at                                  *
+*                                                                          *
+*    http://www.apache.org/licenses/LICENSE-2.0                            *
+*                                                                          *
+* Unless required by applicable law or agreed to in writing, software      *
+* distributed under the License is distributed on an "AS IS" BASIS,        *
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+* See the License for the specific language governing permissions and      *
+* limitations under the License.                                           *
+***************************************************************************/
+
+import Foundation
+
+public class OptimizelyJSON: NSObject {
+    
+    private typealias schemaHandler = (Any) throws -> Void
+    private var payload: String?
+    private var data = [String: Any]()
```

Comment:
What about "map" instead of "data"? data is too generic.

Reply:
following `go` as reference PR, they have named it `data`, i feel `map` makes more sense too.